pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,73 @@
+{
+boost,
+cmake,
+cm256cc,
+fetchFromGitHub,
+fftwFloat,
+glew,
+libav,
+libiio,
+libopus,
+libpulseaudio,
+libusb,
+limesuite,
+mkDerivation,
+ocl-icd,
+opencv3,
+pkgconfig,
+pkgs,
+qtbase,
+qtmultimedia,
+qtwebsockets,
+serialdv,
+stdenv,
+wrapQtAppsHook,
+}:
+
+let
+
+  codec2 = pkgs.codec2.overrideAttrs (old: {
+    src = fetchFromGitHub {
+      owner = "drowe67";
+      repo = "codec2";
+      rev = "567346818c0d4d697773cf66d925fdb031e15668";
+      sha256 = "0ngqlh2cw5grx2lg7xj8baz6p55gfhq4caggxkb4pxlg817pwbpa";
+    };
+  });
+
+in mkDerivation rec {
```

Comment:
```suggestion
in stdenv.mkDerivation rec {
```

Reply:
No i used mkDerivation on purpose, because it is called by libsForQt5.callPackage, this way you don't have to use wrapQtAppsHook
if you read the manual, wrapQtAppsHook is to be used only when you can't use the qt's mkDerivation which we can in that case so it is unneeded ^

Reply:
OK I did overlook that.